content/reference-targets/stm32f769i-discovery.md

```diff
@@ -4,6 +4,104 @@
 
 [Product page](http://www.st.com/en/evaluation-tools/32f769idiscovery.html)
 
+## Connecting and powering the board
+
+There are two USB ports on the board. For firmware flashing and debugging, use the one named "USB ST_LINK"
+![image](https://user-images.githubusercontent.com/71982803/132203876-38002367-37d2-49e0-b6c4-d417352cd6a6.png)
+
+This USB connection will also power the board. Alternatively, it supports PoE, so it may be handy when deploying in the field (although you will not be able to upload new program that way). **To enable PoE, install an appropriate jumper on CN3 header**.
+
+## Installing drivers and ST Link
+
+To access the board, ST Link utility from here (it will also install USB drivers):
+https://www.st.com/en/development-tools/stsw-link004.html#get-software
+
+Unfortunately, ST requires registration to download them. Fortunately, you can use any 10-minute email service to overcome that. For example, https://10minutemail.com/
+
+## Flashing initial nanoFramework firmware
+
+Before you can code in C#, nanoFramework runtime has to be flashed in. This is done by `nanoff` utility, which you installed in the Getting Started guide. Run this command in the Command Prompt:
+
+`nanoff --target ST_STM32F769I_DISCOVERY --update`
+
+This command will download the latest stable FW revision, detect the COM port and flash the board.
```

Comment:
There is no COM port detection going on. With the above command the board it's flashed using the JTAG connection.